hack/docs/internal/multiversion-tutorial/generate_multiversion.go

```diff
@@ -742,9 +742,13 @@ type CronJobStatus struct {
 func (sp *Sample) CodeGen() {
 	cmd := exec.Command("make", "all")
 	_, err := sp.ctx.Run(cmd)
-	hackutils.CheckError("Failed to run make all for cronjob tutorial", err)
+	hackutils.CheckError("Failed to run make all for multiversion tutorial", err)
 
 	cmd = exec.Command("make", "build-installer")
 	_, err = sp.ctx.Run(cmd)
-	hackutils.CheckError("Failed to run make build-installer for  multiversion cronjob tutorial", err)
+	hackutils.CheckError("Failed to run make build-installer for  multiversion tutorial", err)
+
+	cmd = exec.Command("make", "-C", "../../../../../..", "generate-charts")
```

Comment:
```suggestion
	cmd = exec.Command("kubebuilder", "edit", "--plugins=helm.kubebuilder.io/v1-alpha")
```

Here, we need to use the plugin as an end user would do so.